docs/source/fitting/fitfunctions/PEARLTransVoigt.rst

```diff
@@ -0,0 +1,28 @@
+.. _func-PEARLTransVoigt:
+
+===============
+PEARLTransVoigt
+===============
+
+.. index:: PEARLTransVoigt
+
+Description
+-----------
+
+The Voigt profile is used in spectroscopy when a given line shape is neither a Gaussian nor a Lorentzian but rather is a
+convolution of the two. This function is described in Igor Technical Note 26 [1]_. Its relative accuracy is better than
```

Comment:
The reference back to the documentation of an old\legacy piece of data analysis software (Igor) might be a bit weak in that the URL may not be around forever. I had a look into the actual method used and it seems to be based on a paper by someone called Humlícek. So please can you extend this sentence to be something like:
"This function is described in Igor Technical Note 26 [1]_ and is based on a computational method by Humlícek [2]_"